innative/parse.cpp

```diff
@@ -951,3 +969,46 @@ IN_ERROR innative::ParseExportFixup(Module& m, ValidationError*& errors, const E
 
   return ERR_SUCCESS;
 }
+
+IN_ERROR innative::ParseAtomicInstruction(utility::Stream& s, Instruction& ins, const Environment& env)
+{
+  namespace at = innative::atomic_details;
+
+  IN_ERROR err;
+
+  err = ParseByte(s, ins.opcode[1]);
+  if(err < 0)
+    return err;
+
+  uint8_t alignByte;
+  err = ParseByte(s, alignByte);
+  if(err < 0)
+    return err;
+
+  ins.immediates[0]._varuint32 = alignByte & 0b111;
```

Comment:
Why are we assigning a single byte to _varuint32?

Reply:
This was just for symmetry with the existing load/store instructions. They also used the _varuint32 field even though the spec says they can't be bigger than 0x7